pycqed/analysis/analysis_toolbox.py

```diff
@@ -118,47 +118,64 @@ def return_last_n_timestamps(n, contains=''):
 
 
 def latest_data(contains='', older_than=None, newer_than=None, or_equal=False,
-                return_timestamp=False, raise_exc=True,
-                folder=None, return_all=False):
-    '''
-    finds the latest taken data with <contains> in its name.
-    returns the full path of the data directory.
-
-    if older_than is not None, then the latest data that fits and that
-    is older than the date given by the timestamp older_than is returned.
-    if newer_than is not None, than the latest data that fits and that
-    is newer than the date given by the timestamp newer_than is returned
-
-    If no fitting data is found, an exception is raised.
-    Except when you specifically ask not to to
-    this in: raise_exc = False, then a 'False' is returned.
-    return_all = True: returns all the folders that satisfy
-        the requirements (Cristian)
-    '''
-    if (folder is None):
+                return_timestamp=False, return_path=True, raise_exc=True,
```

Comment:
do we really want raise_exc to default to True? I would default it to false and return an empty list if nothing is found. If you explicitely want to raise an error because you're using it in a context where you require outputs then you could activate the flag.